defaults/config.js

```diff
@@ -148,6 +148,14 @@ module.exports = {
 	// @default  null
 	webirc: null,
 
+	//
+	// Storing and reloading messages from sqlite
+	//
+	// @type     boolean
+	// @default  false
+	//
+	sqlite: false,
```

Comment:
I saw your note in PR description, but I think that load-from-backlog is such a desperately wanted feature that it would probably be a good thing to enable it by default.

Because v2.7.0 is starting to be in a good shape, we could target this PR for v3, and therefore the blast radius of setting this by default could be mitigated by the major bump.

No strong opinion, just my 2c.